src/lib/Components/Consignments/Submission/addAssetToConsignment.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  addAssetToConsignmentMutation,
+  addAssetToConsignmentMutationResponse,
+  AddAssetToConsignmentSubmissionInput,
+} from "__generated__/addAssetToConsignmentMutation.graphql"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { commitMutation, graphql } from "relay-runtime"
+
+export const addAssetToConsignment = (input: AddAssetToConsignmentSubmissionInput) => {
+  return new Promise<addAssetToConsignmentMutationResponse>((resolve, reject) => {
+    commitMutation<addAssetToConsignmentMutation>(defaultEnvironment, {
+      mutation: graphql`
+        mutation addAssetToConsignmentMutation($input: AddAssetToConsignmentSubmissionInput!) {
+          addAssetToConsignmentSubmission(input: $input) {
+            asset {
+              submissionID
+            }
+          }
+        }
+      `,
+      variables: {
+        input: {
+          ...input,
+          clientMutationId: Math.random().toString(8),
```

Comment:
Is this needed?

Reply:
I actually don’t believe so, I just wanted to port the existing code with as few changes a possible. I’m open to revisiting, though.

Reply:
Let's leave this as-is for now, next time when I come across it and we don't already have so many changes in flight I'll take a stab.

Reply:
Totally makes sense 👍